### DISCUSSION TITLE
Search in Playlist

### DISCUSSION THREAD

**Discussion by Utarga** [djshaji/WinampSkin#13] [Ideas]
I miss the search option and also the possibility to create a queue.

**Comment by emiliopastran2009-cloud** [comment]
What happened to the project?

**Reply by djshaji** [reply]
I'm writing the whole thing from scratch [here](https://github.com/djshaji/eva). I've already completed the skin loader. You can take it for a spin [here](https://github.com/djshaji/eva/releases/tag/0.1-alpha). It doesn't play music yet. Tap on the winamp icon to load a skin from file. Download skins from https://skins.webamp.org/
